### DISCUSSION TITLE
Free web hosting

### DISCUSSION THREAD

**Discussion by midair55** [VA3HDL/hamdashboard#27] [General]
Visit http://qsl.net for information on obtaining an account on the site to host your dashboard.  Simply upload the three files to the server via FTP once your account is created.   Your dashboard will be available at:  https://qsl.net/callsign   == callsign is all lower case.

**Comment by OK1SLM** [comment]
Tnx
https://www.qsl.net/ok1slm

**Reply by VA3HDL** [reply]
Very nice!
I'll add to the list.
Thanks for sharing it.
73 de Pablo, VA3HDL
